napalm_ios/ios.py

```diff
@@ -27,6 +27,7 @@
 from napalm_base.exceptions import ReplaceConfigException, MergeConfigException
 from napalm_base.utils import py23_compat
 import napalm_base.constants as C
+import napalm_base.helpers
```

Comment:
Why are we calling 'convert' and not just calling helpers.mac()?

I am not seeing any benefit to that (i.e. just adds an unnecessary level of indirection)?